features/step_definitions/common_steps.rb

```diff
@@ -8,6 +8,12 @@
       visit new_user_registration_path
     when "Forgot password page" then
       visit new_user_password_path
+    when "dish list page" then
+      visit dishes_path
+    when "dish detail page" then
+      dish = FactoryGirl.create(:dish, name: 'Dimsun', price: '90',
+                                allergy_info: 'soy', calories: '120')
```

Comment:
Yes. I did it at the beginning.  But then I had the problem - how can a dish be identified in the system? This is the only way I found.